src/classes/PS_CommitmentRequest.cls

```diff
@@ -521,4 +537,21 @@ public inherited sharing class PS_CommitmentRequest {
         public String expirationMonth;
         public String expirationYear;
     }
+
+    /***	
+     * @description Metadata property on the commitment request	
+     */	
+    public class Metadata {	
+        public String originType;	
+        public String originDisplayName;	
+        public String campaignCode;	
```

Comment:
are originDisplayName and campaignCode used anywhere at this point?

Reply:
Not as of this time, but there is a pending work for the Campaign code on create (W-8334258). I can remove attributes, but campaign code is a placeholder for the future work.

Reply:
The unused Metadata properties have been removed, can you please review the PR again? Thanks.